src/main/java/model/QualityIndicator.java

```diff
@@ -0,0 +1,114 @@
+package main.java.model;
+
+import java.util.ArrayList;
+
+/**
+ * A class that represents the indicators quality of the tools in comparison with iPlasma and PDM
+ */
+
+public class QualityIndicator {
+
+	private int PMDDCI, PMDDII, PMDADCI, PMDADII, iPlasmaDCI, iPlasmaDII, iPlasmaADCI, iPlasmaADII;
+	private ArrayList<ExcelRow> excelRows;
+	
+	/**
+	 * Creates a IndicatorQuality object based on:
+	 *
+	 * @param PMDDCI - DCI Indicator - number of faults correctly identified of PDM tool
+	 * @param PMDDII - DII Indicator - number of faults incorrectly indetified of PDM tool
+	 * @param PMDADCI ADCI Indicator - number of absence of faults correctly indetified of PDM tool
+	 * @param PMDADII - ADII Indicator - number of absence of faults incorrectly indetified of PDM tool
+	 * @param iPlasmaDCI - DCI Indicator - number of faults correctly identified of iPlasma tool
+	 * @param iPlasmaDII - DII Indicator - number of faults incorrectly indetified of iPlasma tool
+	 * @param iPlasmaADCI ADCI Indicator - number of absence of faults correctly indetified of iPlasma tool
+	 * @param iPlasmaADII - ADII Indicator - number of absence of faults incorrectly indetified of iPlasma tool
+	 * 
+	 * A IndicatorQuality object also compute the Quality Indicators 
```

Comment:
what?

Reply:
Qual é a melhor forma de escrever em inglês "O objeto IndicatorQuality também calcula os inidicadores de qualidade"? Era este o meu objetivo ahahaha

Reply:
Acho que fica melhor assim "A IndicatorQuality object also calculate the Quality Indicators from the Excel file"